ui/app/components/pending-tx.js

```diff
@@ -15,7 +15,7 @@ const addressSummary = util.addressSummary
 const nameForAddress = require('../../lib/contract-namer')
 const BNInput = require('./bn-as-decimal-input')
 
-const MIN_GAS_PRICE_GWEI_BN = new BN(1)
```

Comment:
No, sorry, when I copied the `responsive-ui` folder over `ui` the merge seems to be prioritizing the `NewUI` over `master`, so it's changing some things like this.  Good catch, I'll review the diffs for other things like this.